### PR TITLE
adding rtl theme

### LINKED ISSUE #704: [Refactor] Floating label not changing direction
Blocked by #843 

1. Go to home page
2. Log in
3. Click "פרטי משתמש" while on Hebrew language
  - see the floating label is floating on the top left side of the input - it should change direction to the right

image

### PR DESCRIPTION
**Pull request by EyalIlan**
closes #704

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -31,32 +32,33 @@ const App: FC = () => {
 
   const appDir = i18n.dir();
   useEffect(() => {
-    // https://material-ui.com/guides/right-to-left/
+    // https://material-ui.com/guides/right-to-left/F
     document.body.dir = appDir;
-    theme.direction = appDir;
-  }, [i18n, theme, theme.direction, appDir]);
+    // theme.direction = appDir;
+    // , theme.direction
```
Comment (SejoB) [inline src/App.tsx:38]: remove comments

File: src/index.tsx [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
 import OverlayLoader from './components/molecules/OverlayLoader';
 
+
+const jss = create({ plugins: [...jssPreset().plugins, rtl()] });
+
 ReactDOM.render(
+  
   <Suspense fallback={<OverlayLoader show />}>
+     <StylesProvider jss={jss}>
     <App />
```
Comment (SejoB) [inline src/index.tsx:19]: add tab

File: src/App.tsx [hunk 3]
```diff
@@ -15,14 +16,14 @@ const headerHeight = '5vh';
 const pageContentHeight = '88vh';
 const footerHeight = '7vh';
 
+
```
Comment (SejoB) [inline src/App.tsx:19]: remove space

File: src/App.tsx [hunk 4]
```diff
@@ -1,4 +1,5 @@
 import React, { FC, useEffect } from 'react';
+
```
Comment (SejoB) [inline src/App.tsx:2]: remove space

**Review by SejoB — CHANGES_REQUESTED** [review]

**Comment by SejoB** [comment]
also don't forget assign yourself to PR

File: src/App.tsx [hunk 5]
```diff
@@ -31,32 +32,33 @@ const App: FC = () => {
...
   const appDir = i18n.dir();
   useEffect(() => {
-    // https://material-ui.com/guides/right-to-left/
+    // https://material-ui.com/guides/right-to-left/F
     document.body.dir = appDir;
-    theme.direction = appDir;
-  }, [i18n, theme, theme.direction, appDir]);
+    // theme.direction = appDir;
+    // , theme.direction
+  }, [i18n, theme, appDir]);
```
Comment (SejoB) [inline src/App.tsx:39]: remove comments

**Review by SejoB — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 6]
```diff
@@ -31,32 +32,33 @@ const App: FC = () => {
 
   const appDir = i18n.dir();
   useEffect(() => {
-    // https://material-ui.com/guides/right-to-left/
+    // https://material-ui.com/guides/right-to-left/F
```
Comment (yuvalbl) [inline src/App.tsx:35]: remove F

**Pull request closed without merging**